Mail/Utils/ShortcutModifier.swift

```diff
@@ -0,0 +1,131 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import InfomaniakCoreUI
+import InfomaniakDI
+import MailCore
+import MailResources
+import SwiftUI
+
+struct ShortcutModifier: ViewModifier {
+    @EnvironmentObject private var actionsManager: ActionsManager
+    @EnvironmentObject private var navigationState: NavigationState
+
+    @LazyInjectService private var matomo: MatomoUtils
+    @LazyInjectService private var platformDetector: PlatformDetectable
+
+    @ObservedObject var viewModel: ThreadListViewModel
+    @ObservedObject var multipleSelectionViewModel: ThreadListMultipleSelectionViewModel
+
+    func body(content: Content) -> some View {
+        ZStack {
+            VStack {
+                Button(MailResourcesStrings.Localizable.actionDelete, action: shortcutDelete)
+                    .keyboardShortcut(.delete, modifiers: [])
+
+                Button(MailResourcesStrings.Localizable.actionReply, action: shortcutReply)
+                    .keyboardShortcut("r")
+
+                Button(MailResourcesStrings.Localizable.buttonNewMessage, action: shortcutNewMessage)
```

Comment:
I understand that you put it here to have all the shortcuts in the same place but I think it would be more relevant to have this one attached to the compose button. What do you think ?

Reply:
Good idea for this one.
I'll try to add it to the real button